jenkins-agent.yaml

```diff
@@ -5,7 +5,7 @@ spec:
   serviceAccountName: jenkins-agent
   containers:
     - name: git
-      image: alpine/git
+      image: bitnami/git
```

Comment:
I was getting really frustrated with the idiosyncrasies between `sh` and `bash`, but I might have gotten around that anyways.  I'll try again with the alpine image.

I also did switch from `wget` to `curl` which is bundled in the `bitnami/git` image.  it might be best to create a purpose built container for this anyways.